src/main/java/menion/android/whereyougo/VersionInfo.java

```diff
@@ -31,11 +31,9 @@
  */
 public class VersionInfo {
 
-    // private static final String TAG = "MainAfterStart";
-
-    /****************************************/
-  /* AFTER START ACTIONS */
-    /****************************************/
+    /*
+     * AFTER START ACTIONS
+    */
```

Comment:
Maybe change to `/* AFTER START ACTIONS */` like in the other classes?

Reply:
Reasonable. Will do so in the force push.